receiver/apachereceiver/internal/metadata/deprecated_metrics.go

```diff
@@ -0,0 +1,256 @@
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Gauge().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetDoubleValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+}
+
+func (m *metricApacheLoad5) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val float64, serverNameAttributeValue string) {
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Gauge().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetDoubleValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+}
+
+func (m *metricApacheRequestTime) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string) {
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Sum().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetIntValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+}
+
+func (m *metricApacheRequests) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string) {
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Sum().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetIntValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+}
+
+func (m *metricApacheScoreboard) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string, scoreboardStateAttributeValue string) {
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Sum().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetIntValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+	dp.Attributes().PutStr("state", scoreboardStateAttributeValue)
+}
+
+func (m *metricApacheTraffic) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string) {
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Sum().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetIntValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+}
+
+func (m *metricApacheUptime) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string) {
+	if !m.settings.Enabled {
+		return
+	}
+	dp := m.data.Sum().DataPoints().AppendEmpty()
+	dp.SetStartTimestamp(start)
+	dp.SetTimestamp(ts)
+	dp.SetIntValue(val)
+	dp.Attributes().PutStr("server_name", serverNameAttributeValue)
+}
+
+func (m *metricApacheWorkers) RecordApacheLoad1DataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string, workersStateAttributeValue string) {
```

Comment:
Nit: I believe `1` isn't needed, right?
```suggestion
func (m *metricApacheWorkers) RecordApacheLoadDataPointWithServerName(start pcommon.Timestamp, ts pcommon.Timestamp, val int64, serverNameAttributeValue string, workersStateAttributeValue string) {
```

Reply:
Actually, the whole substring `ApacheLoad1` is redundant - I guess that VSC changed it when I was changing all occurrences of this function and I didn't notice, sorry.